Caculator_G/Caculator_G/ViewController.swift

```diff
@@ -11,21 +11,85 @@ import UIKit
 class ViewController: UIViewController {
     
     @IBOutlet weak var numLabel: UILabel!
+    var labelFontSize : CGFloat = 70
+    var numValueText : String = "0"
+    var numDisplayText : String = "0"
+    var isNegative : Bool = false
```

Comment:
```suggestion
    var numValueText = "0"
    var numDisplayText = "0"
    var isNegative = false
```

CGFloat은 Double 등과 혼동하게 쉬워서 타입을 명시하는게 좋지만, 다른 타입들은 Swift의 타입추론을 믿고, 생략 가능해요

Reply:
영 질문있습니다 !! 
코드의 규모가 커지게 되는 경우 컴파일이 느려지는걸 최대한 막기위해 시도하는 방법중에 하나로 아래의 글을 읽은 적이 있습니다.
타입을 명시해주지 않으면 타입체킹하는데 에너지를 소모하는게 있어서 타입 명시를 통해 컴파일 속도(?)를 향상시킨다는 글을 읽은 적이 있는 것 같은데 이게 확실한 정보가 맞나요?